### PR TITLE
Uniform results data

### PR DESCRIPTION
**Pull request by acomodi**
This PR makes results more uniform and creates a true DataFrame that can be queried to extract information on the runs.

### CODE REVIEW

**Comment by acomodi** [comment]
@HackerFoo @litghost @mithro  FYI. [This is the colab page](https://colab.research.google.com/drive/1ShBE2FrXEh33LD01e6hh4yM0H9p4sHYM?authuser=1#scrollTo=jk_IBrHiop-A&uniqifier=3) that uses the dataframe produced with this PR.

File: dataframe.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
```
Comment (kgugala) [inline dataframe.py:1]: License header is missing here

File: symbiflow.py [hunk 2]
```diff
@@ -443,13 +447,13 @@ def resources(self):
...
-            "GLB": "unsupported",
-            "PLL": "NA",  #str(pll),
-            "IOB": "NA",  #str(iob),
+            "LUT": None,  #str(lut),
+            "DFF": None,  #str(dff),
+            "BRAM": None,  #str(bram),
+            "CARRY": None,  #str(carry),
+            "GLB": None,
+            "PLL": None,  #str(pll),
+            "IOB": None,  #str(iob),
```
Comment (kgugala) [inline symbiflow.py:456]: can we remove those comments?

File: dataframe.py (hunk 1 above)
Comment (acomodi) [reply dataframe.py:1]: Done

File: symbiflow.py (hunk 2 above)
Comment (acomodi) [reply symbiflow.py]: Done

**Pull request merged**